Task_2_2_1/src/main/java/r/nsu/fit/masud/Main.java

```diff
@@ -0,0 +1,70 @@
+package r.nsu.fit.masud;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.JsonArray;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonParser;
+
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Main {
```

Comment:
![image](https://user-images.githubusercontent.com/25470436/232706954-57a220af-e369-4848-a547-3b6e4d4ebf00.png)

In this PR you should only have Task 2_2_1 directory

Reply:
understood